### DISCUSSION TITLE
Downloads Page

### DISCUSSION THREAD

**Discussion by LWFlouisa** [olive-editor/olive#2351] [General]
Was the version of Olive Editor version 1? Why is it missing? I was able to download that just a few days ago.

**Comment by Simran-B** [comment]
The only existing versions are the 0.1.0, 0.1.1, and 0.1.2 alphas as well as the unstable 0.2.0 builds.
Preserved 0.1.0 and 0.2.0 builds are available here:
- https://github.com/olive-editor/olive/releases/tag/0.1.0
- https://github.com/olive-editor/olive/releases/tag/0.2.0-nightly
